bears/python/requirements/PySafetyBear.py

```diff
@@ -77,7 +77,7 @@ def run(self, filename, file):
             yield Result(
                 self,
                 message_template.format(vuln=vulnerability),
-                additional_info=vulnerability.description,
+                additional_info=vulnerability.data['advisory'],
```

Comment:
wtf, no.  This makes it so that it will not work with the old advisory data.

The `try, except` from the last code review ( https://github.com/coala/coala-bears/pull/2095#discussion_r147941937 ) is mandatory.

Reply:
Hmmm, I've checked safety 0.5.1, and it [always](https://github.com/pyupio/safety/blob/1380740696100a5724c4f33c735df9364de77ddf/safety/safety.py#L50) fetches new data, so this change is likely to be safe.